phpcs.xml

```diff
@@ -20,6 +20,7 @@
     <exclude-pattern>*/tests/_support/_generated/*</exclude-pattern>
     <exclude-pattern>*/tests/_helpers/*</exclude-pattern>
     <exclude-pattern>*/tests/_output/*</exclude-pattern>
+    <exclude-pattern>*/tests/*/_data/*</exclude-pattern>
```

Comment:
Wouldnt we need this first in suite-nonsplit? Same for other changes?
We cannot just change it here via merge to avoid conflicts IMO.
You can test it and then we need to introduce on the other side.

Reply:
Done via https://github.com/spryker-shop/suite/commit/86f46223884136c9fc2e567b9dc71414fc5802dc as this was critical!